user-interface/src/lib/components/cams/RichTextEditor/editor.constants.ts

```diff
@@ -0,0 +1,5 @@
+export const ZERO_WIDTH_SPACE = '​';
+export const ZERO_WIDTH_SPACE_REGEX = new RegExp(ZERO_WIDTH_SPACE, 'g');
+export const EMPTY_TAG_REGEX = /<([a-z][^>]*)>\s*(?:<br\s*\/?>)*\s*<\/\1>/gi;
+
+export type RichTextFormat = 'strong' | 'em' | 'u';
```

Comment:
We should double-check that each of these constants are actually used and not redefined elsewhere unnecessarily.